jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -642,6 +642,7 @@
           $custom_settings
 
           perl -e "alarm 6*60*60 ; exec '${mkcloudwrapper} bash ${automationrepo}/scripts/mkcloud setuphost $(echo -n $MKCLOUDTARGET) ' ; print qq{$!\n} ; exit 127 " | tee $artifacts_dir/mkcloud_short_stdout.log
+          ${automationrepo}/scripts/timeit.py $artifacts_dir/mkcloud_short_stdout.log | tee $artifacts_dir/cistats.txt
           ret=${PIPESTATUS[0]}
```

Comment:
you should add it one line down to not screw the return code

Reply:
sorry